### PR TITLE
Added tapping circles

### PR DESCRIPTION
**Pull request by paoloricciuti**

### CODE REVIEW

**Review by DeveloperKhan — COMMENTED** [review]
looks good, doesn't seem to prevent additional clicks. Would you be able to also trigger the tapping circle when a key is registered (like pressing the space button makes the animation on the center of the screen)

**Comment by paoloricciuti** [comment]
By "doesn't seem to prevent additional clicks" do you mean it should prevent additional clicks? Also I've thought about triggering it on space-bar but the space-bar could also be keep pressed.

I may be able to take a look and trigger the "tap" animation in the function that sends the tap to the backend (this would probably require saving the last clicked point). Also this would show the tapping animation only when the move actually gets sent to the server. Is that what we want?

**Comment by DeveloperKhan** [comment]
> By "doesn't seem to prevent additional clicks" do you mean it should prevent additional clicks? Also I've thought about triggering it on space-bar but the space-bar could also be keep pressed.
> 
> I may be able to take a look and trigger the "tap" animation in the function that sends the tap to the backend (this would probably require saving the last clicked point). Also this would show the tapping animation only when the move actually gets sent to the server. Is that what we want?

For the first question, I just meant I'm happy with the behavior. I'm glad you can "click through" the animation. I'd prefer if none of this animation logic hits the backend, I don't think it is necessary. For the case of holding the spacebar, just trigger the animation every turn so it matches the fast move animation. Otherwise, trigger the animation at the correct spot (either in the bottom-center of the screen, on the charge move buttons, or on the switch buttons) on "keyboard up."

**Comment by paoloricciuti** [comment]
> > By "doesn't seem to prevent additional clicks" do you mean it should prevent additional clicks? Also I've thought about triggering it on space-bar but the space-bar could also be keep pressed.
> > I may be able to take a look and trigger the "tap" animation in the function that sends the tap to the backend (this would probably require saving the last clicked point). Also this would show the tapping animation only when the move actually gets sent to the server. Is that what we want?
> 
> For the first question, I just meant I'm happy with the behavior. I'm glad you can "click through" the animation. I'd prefer if none of this animation logic hits the backend, I don't think it is necessary. For the case of holding the spacebar, just trigger the animation every turn so it matches the fast move animation. Otherwise, trigger the animation at the correct spot (either in the bottom-center of the screen, on the charge move buttons, or on the switch buttons) on "keyboard up."

Yeah what I meant is that on the fast move turn I could trigger the animation. The issue could be that is gets triggered two times when you use the mouse. But I'm saying this without looking at the code. Tomorrow I'll take a look and modify the code appropriately.

File: src/components/team_selector/style.module.scss [hunk 1]
```diff
@@ -28,20 +28,20 @@
 	text-align: left;
 	overflow-y: scroll;
 	margin: auto;
-    z-index: 99991;
-    background-color: white;
+	z-index: 99991;
+	background-color: rgba(255, 255, 255, 1);
```
Comment (DeveloperKhan) [inline src/components/team_selector/style.module.scss:32]: strange u have to do this, I'll look into it

**Review by DeveloperKhan — APPROVED** [review]

**Pull request merged**